### DISCUSSION TITLE
[Projects Beta] should be able to paste images from clipboard

### DISCUSSION THREAD

**Discussion by joshstiksma** [community/community#14920] [Projects and Issues]
Many issue tracking packages allow for quickly attaching files/images via copy/paste or drag and drop. Screenshots of issues would be ideal for me to consider using projects.

**Comment by Sammeeey** [comment]
### This feedback is a duplicate of #9917
@joshstiksma maybe you could edit your initial feedback to inlcude a link to #9917 ?
Then we would have a higher chance of succes, if @davidjb99 improves the description there?
### Visualization of issue
- manual insertion of images (with URLs) possible
- copy+paste insertion (without URLs) impossible
![grafik](https://user-images.githubusercontent.com/49591562/164958924-d23696a4-f241-4b56-9525-dd5e2a47f007.png)
![grafik](https://user-images.githubusercontent.com/49591562/164959033-58f1b71b-dd9a-4797-b7b4-9f0a8e63f2bc.png)


### Why is this an issue?
Sometimes you just have images/screenshots in the clipboard (which don't have URL's).
Hence we need a solution to insert these images too✌
